src/main/java/net/jung/chess/Board.java

```diff
@@ -1,132 +1,142 @@
-        addWhitePiece(Piece.createWhiteKing());
-        addWhitePiece(Piece.createWhiteBishop());
-        addWhitePiece(Piece.createWhiteKnight());
-        addWhitePiece(Piece.createWhiteRook());
+        points -= pawnSizeInFile(color) * 0.5;
+        return points;
     }
 
+    public List<Piece> getPieceListByColor(Piece.Color color) {
+        List<Piece> colorPieceList = new ArrayList<>();
 
-    public String getWhitePiecesRepresentation() {
-        return getPiecesRepresentation(whitePieceList);
+        for (Rank rank : rankList) {
+            for (int i = 0; i < Rank.MAX_PIECES_IN_RANK; i++) {
+                Piece piece = rank.getPiece(i);
+                if (piece.getColor() == color) {
+                    colorPieceList.add(piece);
+                }
+            }
+        }
+        return colorPieceList;
     }
 
-    public String getBlackPiecesRepresentation() {
-        return getPiecesRepresentation(blackPieceList);
-    }
+    public int pawnSizeInFile(Piece.Color color) {
+        int[] pawnsInFile = new int[8];
+        int excessPawnsInFile = 0;
 
-    public String getPiecesRepresentation(List<Piece> pieceList) {
-        StringBuilder piecesRepresentation = new StringBuilder();
-        for (int i = 0; i < pieceList.size(); i++) {
-            piecesRepresentation.append(pieceList.get(i).getRepresentation());
-            if(i==7) {
-                piecesRepresentation.append(System.getProperty("line.separator"));
+        for (Rank rank : rankList) {
+            for (int i = 0; i < Rank.MAX_PIECES_IN_RANK; i++) {
+                Piece piece = rank.getPiece(i);
+                pawnsInFile[i] += piece.isColorType(color, Piece.Type.PAWN) ? 1 : 0;
             }
         }
-        return piecesRepresentation.toString();
-    }
 
-    public String boardLayoutToString() {
+        for (int pawnSize : pawnsInFile) {
+            if(pawnSize>1) {
+                excessPawnsInFile += pawnSize ;
+            }
+        }
 
-        StringBuilder boardLayout = new StringBuilder();
-        boardLayout
-                .append(appendNewLine(getBlackPiecesRepresentation()))
-                .append(BLANK_RANK)
-                .append(BLANK_RANK)
-                .append(BLANK_RANK)
-                .append(BLANK_RANK)
-                .append(appendNewLine(getWhitePiecesRepresentation()));
-        return boardLayout.toString();
+        return excessPawnsInFile;
     }
 
-    public void reset() {
-        whitePieceList.clear();
-        blackPieceList.clear();
+    public List<Piece> sortColorPiecesAscending(Piece.Color color) {
+        List<Piece> colorPieceList = getPieceListByColor(color);
+        Collections.sort(colorPieceList);
+        return colorPieceList;
     }
 
+    public List<Piece> sortColorPiecesDescending(Piece.Color color) {
+        List<Piece> colorPieceList = getPieceListByColor(color);
+        Collections.sort(colorPieceList, Collections.reverseOrder());
```

Comment:
```suggestion
        colorPieceList.sort(Collections.reverseOrder());
```
List에도 sort라는 내장 함수가 있습니다.